VENDORING.md

```diff
@@ -0,0 +1,55 @@
+# Vendoring policies
+
+## Vendoring using tags
+
+Commit ID based vendoring provides little/no information about the updates
+vendored. To fix this, vendors will now require that repositories use annotated
+tags to snapshot commits.
+
+Each tag should:
+- Follow Semantic Versioning rules (refer to section on "Semantic Versioning")
+- Have a corresponding entry in the repo's CHANGELOG.md that describes changes
+since the previous update.
+
+Each repo should:
+- Have a mandatory CHANGELOG.md at the root of the repo.
+
+The goal here is for consuming repos to be able to use the tag version and
+changelog updates to determine whether the vendoring will cause any  breaking or
+backward incompatible changes. This also means that repos can specify having
+dependency on a package of a specific version or greater up to the next major
+release, without encountering breaking changes.
+
+## Semantic Versioning
```

Comment:
I don't think it's necessary or even correct to mandate a versioning policy for other projects, especially since we don't really follow semver either.